services/core-api/app/api/exports/static_content/resources/core_static_content_resource.py

```diff
@@ -74,7 +76,8 @@ class StaticContentResource(Resource):
     )
     @requires_role_view_all
     def get(self):
-        content_json = cache.get(STATIC_CONTENT_KEY)
+        # content_json = cache.get(STATIC_CONTENT_KEY)
+        content_json = None
         if not content_json:
```

Comment:
Is this a placeholder?

Reply:
I'm going to guess debug code form Justin, I'll revert.